gulpfile.js

```diff
@@ -219,6 +219,16 @@ const mappings = {
     'dir': 'powerbiembedded/lib',
     'source': 'arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json'
   },
+  'rdbms.mysql': {
+    'packageName': 'azure-arm-rdbms',
```

Comment:
Yes. afaik, there's a single resource provider that handles both of these, so this seems the most correct approach.